YIF.Core.Service/Concrete/Services/ValidatorServices/ChangePasswordValidator.cs

```diff
@@ -25,6 +25,9 @@ public ChangePasswordValidator(UserManager<DbUser> userManager, IRecaptchaServic
                .NotNull().WithMessage("Recaptcha є обов'язковою!")
                .Must(_recaptcha.IsValid).WithMessage("Роботи атакують!");
 
+            RuleFor(x => x.UserId)
```

Comment:
I know that you didn't write that code, but could you change this hardcoded strings (WithMessage("blabla")) to resourceManager?
![image](https://user-images.githubusercontent.com/73486410/120110871-18213d80-c178-11eb-8d53-118193415d58.png)